dobi.yaml

```diff
@@ -209,3 +209,18 @@ alias=release:
       - 'github-release'
       - 'release-version:capture(DOBI_VERSION)'
       - 'dist-img:push'
+
+
+
+image=dobi:
+    image: test
+    tags: ["latest"]
+    context: .
+#    dockerfile: Dockerfile
+    dobifile:
+      - FROM: busybox
+      - RUN: ls
+      - RUN: mkdir {env.HOME:}
+      - RUN: env
+      - CMD: pwd
```

Comment:
I though it would be easier to validade if a key matches a existing Docker command and supply custom ones. Plus it keeps the yaml with nice colors, and a slice would still require the initial dash anyway, so it would never be that a straight foward copy/paste. Its your call
![dobi](https://cloud.githubusercontent.com/assets/6259987/20012876/348f3aac-a298-11e6-9998-8fc47014812a.png)